fastlane/templates/SalemoveSDK.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
```

Comment:
Does it make sense to add comment to this template that resulting pod spec is auto-generated file, and should not be edited directly, but through this template?

Reply:
That would mean that every file generated from this one would also have the same comment, which would be a bit confusing, as the text would be the same on both. Maybe it would be better for me to add a README instead with instructions and observations?